README.md

```diff
@@ -10,58 +10,33 @@ The definition of this GitHub Action is in [action.yml](https://github.com/Azure
+
+If you are have extension project(s) in your repo, these templates will **NOT** resolve the **extensions.csproj** in your project. If you want to use binding extensions (e.g. Blob/Queue/EventHub Triggers), please consider [registering Azure Functions binding extensions](https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-register) in your host.json.
+
+Alternatively, you can add a `- run: dotnet build --output ./bin` step **before** functions-action step.
+
+## Using Publish Profile as Deployment Credential (recommended)
+You may want to get the publish profile from your function app.
+Using publish profile as deployemnt credential is recommended if you are not the owner of your Azure subscription.
 
-## Using Publish Profile as Deployment Credential
-You may want to get the publish profile from your function app. Using publish profile as deployemnt credential is recommended
-if you are not the owner of your Azure subscription.
 1. In Azure portal, go to your function app.
 2. Click **Get publish profile** and download **.PublishSettings** file.
 3. Open the **.PublishSettings** file and copy the content.
-4. Paste the XML content to your Github Repository > Settings > Secrets > Add a new secret > **SCM_CREDENTIALS**
-
-## Create Azure function app and Deploy using GitHub Actions (Publish Profile)
-1. Follow the tutorial [Azure Functions Quickstart](https://docs.microsoft.com/en-us/azure/azure-functions/functions-create-first-function-vs-code)
-2. Use the following template to create the `.github/workflows/` file in your project repository.
-3. Change `app-name` to your function app name.
-4. Commit and push your project to GitHub repository, you should see a new GitHub workflow initiated in **Actions** tab.
-```yaml
-name: Linux_Node_Workflow_ScmCred
-
-on:
-  push:
-    branches:
-    - master
-
-jobs:
-  build-and-deploy:
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@master
-    - uses: actions/setup-node@v1
-      with:
-        node-version: '10.x'
-    - name: 'run npm'
-      run: |
-        npm install
-        npm run build --if-present
-        npm run test --if-present
-    - uses: Azure/functions-action@v1
-      id: fa
-      with:
-        app-name: PLEASE_REPLACE_THIS_WITH_YOUR_FUNCTION_APP_NAME
-        publish-profile: ${{ secrets.SCM_CREDENTIALS }}
+4. Paste the XML content to your Github Repository > Settings > Secrets > Add a new secret > **AZURE_FUNCTIONAPP_PUBLISH_PROFILE**
+5. Use the above template to create the `.github/workflows/your-workflow.yml` file in your project repository.
+6. Change variable values in `env:` section according to your function app.
+7. Commit and push your project to GitHub repository, you should see a new GitHub workflow initiated in **Actions** tab.
 
-```
-
-## Using Azure Service Principle for RBAC as Deployment Credential
+## Using Azure Service Principle for RBAC as Deployment Credential (deprecated)

```

Comment:
I found this method simple to implement. Is there information anywhere on why this was deprecated? Will the publish profile be the only supported method going forward? [The sample repo](https://github.com/Azure/actions-workflow-samples/tree/master/FunctionApp) also needs to be updated with this information.